### ISSUE TITLE
Libreswan 4.13 is not building on Ubuntu 20.04

### ISSUE THREAD

**Issue by Joel-Robinson-Redox** [libreswan/libreswan#1643]
Builds of 4.13 are failing on our build server with the following error. 
```bash
/home/ssm-user/libreswan413/libreswan-4.13/programs/pluto/x509.c: In function ‘gntoid’:
/home/ssm-user/libreswan413/libreswan-4.13/programs/pluto/x509.c:349:13: error: variable ‘i’ has initializer but incomplete type
  349 |   for (enum ip_index i = 0; i < IP_INDEX_ROOF; i++) {
      |             ^~~~~~~~
/home/ssm-user/libreswan413/libreswan-4.13/programs/pluto/x509.c:349:22: error: storage size of ‘i’ isn’t known
  349 |   for (enum ip_index i = 0; i < IP_INDEX_ROOF; i++) {
      |                      ^
/home/ssm-user/libreswan413/libreswan-4.13/programs/pluto/x509.c:349:3: error: ‘enum ip_index’ declared in ‘for’ loop initial declaration
  349 |   for (enum ip_index i = 0; i < IP_INDEX_ROOF; i++) {
      |   ^~~
/home/ssm-user/libreswan413/libreswan-4.13/programs/pluto/x509.c:349:33: error: ‘IP_INDEX_ROOF’ undeclared (first use in this function)
  349 |   for (enum ip_index i = 0; i < IP_INDEX_ROOF; i++) {
      |                                 ^~~~~~~~~~~~~
/home/ssm-user/libreswan413/libreswan-4.13/programs/pluto/x509.c:349:33: note: each undeclared identifier is reported only once for each function it appears in
/home/ssm-user/libreswan413/libreswan-4.13/programs/pluto/x509.c:350:8: error: ‘ip_families’ undeclared (first use in this function)
  350 |    if (ip_families[i].ip_size == gn->name.len) {
      |        ^~~~~~~~~~~
/home/ssm-user/libreswan413/libreswan-4.13/programs/pluto/x509.c:349:22: error: unused variable ‘i’ [-Werror=unused-variable]
  349 |   for (enum ip_index i = 0; i < IP_INDEX_ROOF; i++) {
      |                      ^
cc1: all warnings being treated as errors
make[5]: *** [../../mk/rules.mk:63: x509.o] Error 1
make[5]: *** Waiting for unfinished jobs....
make[4]: *** [../../mk/targets.mk:82: all] Error 2
make[4]: Leaving directory '/home/ssm-user/libreswan413/libreswan-4.13/programs/pluto'
make[3]: *** [../mk/targets.mk:82: recursive-all] Error 2
make[3]: Leaving directory '/home/ssm-user/libreswan413/libreswan-4.13/programs'
make[2]: *** [/home/ssm-user/libreswan413/libreswan-4.13/mk/targets.mk:82: recursive-all] Error 2
make[2]: Leaving directory '/home/ssm-user/libreswan413/libreswan-4.13'
dh_auto_build: error: make -j2 programs ARCH=amd64 IPSECVERSION=4.13-1 PREFIX=/usr FINALLIBEXECDIR=/usr/libexec/ipsec FINALMANDIR=/usr/share/man USE_LDAP=true USE_LIBCAP_NG=true USE_LABELED_IPSEC=true USE_LINUX_AUDIT=true returned exit code 2
make[1]: *** [debian/rules:23: override_dh_auto_build] Error 25
make[1]: Leaving directory '/home/ssm-user/libreswan413/libreswan-4.13'
make: *** [debian/rules:6: binary] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
debuild: fatal error at line 1182:
dpkg-buildpackage -us -uc -ui -i -b failed
make[1]: *** [Makefile:7: deb] Error 29
make[1]: Leaving directory '/home/ssm-user/libreswan413/libreswan-4.13/packaging/debian'
make: *** [Makefile:184: deb] Error 2
```
Builds were working for 4.13~rc1, so it's a pretty recent change. 
`gcc` info:
```bash
% gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/9/lto-wrapper
OFFLOAD_TARGET_NAMES=nvptx-none:hsa
OFFLOAD_TARGET_DEFAULT=1
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 9.4.0-1ubuntu1~20.04.2' --with-bugurl=file:///usr/share/doc/gcc-9/README.Bugs --enable-languages=c,ada,c++,go,brig,d,fortran,objc,obj-c++,gm2 --prefix=/usr --with-gcc-major-version-only --program-suffix=-9 --program-prefix=x86_64-linux-gnu- --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --libdir=/usr/lib --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --with-default-libstdcxx-abi=new --enable-gnu-unique-object --disable-vtable-verify --enable-plugin --enable-default-pie --with-system-zlib --with-target-system-zlib=auto --enable-objc-gc=auto --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --enable-multilib --with-tune=generic --enable-offload-targets=nvptx-none=/build/gcc-9-9QDOt0/gcc-9-9.4.0/debian/tmp-nvptx/usr,hsa --without-cuda-driver --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 9.4.0 (Ubuntu 9.4.0-1ubuntu1~20.04.2)
```
Looks like it's relates to some [x509 fix that just got merged](https://github.com/libreswan/libreswan/commit/474d75be074799efa9e38f346d3fdb76dec3eead) to `v4_maint`.

**Comment by cagney** [comment]
Yea,  see #1644

**Comment by paulwouters** [comment]
Resolved in v4.14 that was just released to fix this specific issue. Our apologies.
